libs/entry-components/styles/modules/components/checkboxes/_generator.scss

```diff
@@ -0,0 +1,27 @@
+@use 'sass:map';
+
+/* stylelint-disable-next-line max-line-length */
+$checkbox-selector: '.mdc-checkbox__native-control:enabled:not(:checked):not(:indeterminate):not([data-indeterminate = "true"]) ~ .mdc-checkbox__background';
+
+@mixin generate-from($theme) {
+	$background-color: map.get($theme, 'general', 'checkboxes', 'background');
```

Comment:
Maybe we could consider renaming variable to $background since the context seems clear enough.